scripts/translate_schemas.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+import sys
+import subprocess
+import docker
```

Comment:
```suggestion

import docker
```

Reply:
I'm confused as to why this should block the PR?

Reply:
Imports should be grouped in the following order:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

Reply:
Oh right, it's part of PEP8 suggestions, but this isn't something that any of our linters check.

Reply:
This has been updated in a previous commit.

Reply:
I don't see any change to the file?

Reply:
Don't know why that would be - was changed in this commit: https://github.com/ONSdigital/eq-survey-runner/pull/2213/commits/d2a80077f3856a9475117b701c07bfcea2a91270